runtimevar/driver/driver.go

```diff
@@ -39,32 +41,25 @@ type Variable struct {
 // dictate the type of Variable.Value and a decoding function.  The Watcher provider can use the
 // runtimevar.Decoder to facilitate the decoding logic.
 type Watcher interface {
-	// WatchVariable returns one of:
-	//
-	// 1. A new value for the variable in v, along with a provider-specific
-	//    version that will be passed to the next WatchVariable call. wait is
-	//    ignored and err must be nil.
-	// 2. A new error. v, version, and wait are ignored.
-	// 3. A nil v and version and err, indicating that the value of the variable
-	//    or the error returned has not changed. WatchVariable will not be called
-	//    again for wait.
+	// WatchVariable returns the current State of the variable.
+	// If the State has not changed, it returns nil plus a wait time (which may
+	// be 0); WatchVariable will not be called again for wait.
```

Comment:
...will not be called again for the wait time.

Reply:
Done.